src/authentication/validate-invite/index.tsx

```diff
@@ -69,48 +68,49 @@ export class Invite extends React.Component<Properties, State> {
         errorMessage = 'Invite code error.';
     }
 
-    return <Alert variant='error'>{errorMessage}</Alert>;
+    return (
+      <Alert className={c('alert')} variant='error'>
+        {errorMessage}
+      </Alert>
+    );
   };
 
   render() {
     return (
       <div className={c('')}>
-        <h3 className={c('heading')}>Add your invite code</h3>
-        <div className={c('sub-heading')}>This is the 6 digit code you received in your invite</div>
-
+        <div className={c('heading-container')}>
+          <h3 className={c('heading')}>Add invite code</h3>
+          <div className={c('sub-heading')}>6 digit code you received in your invite</div>
+        </div>
         <form className={c('form')} onSubmit={this.submitForm}>
-          <Input
-            onChange={this.onInviteCodeChanged}
-            placeholder='E.g. 123456'
-            value={this.state.inviteCode}
-            type='text'
-          />
-
-          {this.state.renderAlert &&
-            this.props.inviteCodeStatus !== InviteCodeStatus.VALID &&
-            this.renderAlert(this.props.inviteCodeStatus)}
+          <div className={c('input-container')}>
+            <Input
+              onChange={this.onInviteCodeChanged}
+              placeholder='e.g 123456'
+              value={this.state.inviteCode}
+              type='text'
+              error={this.state.renderAlert}
+            />
+
+            {this.state.renderAlert &&
+              this.props.inviteCodeStatus !== InviteCodeStatus.VALID &&
+              this.renderAlert(this.props.inviteCodeStatus)}
+          </div>
 
           <Button
+            className={c('submit-button')}
```

Comment:
Is there a reason to add this class? Doesn't seem like we're using it anywhere.

Reply:
Ah, I will be using it in the follow up PRs for this story, but not using in this PR.

It will replace the `> *:last-child {}` here in `authentication/_styles.scss`

```
@mixin form {
  display: flex;
  flex-direction: column;
  align-items: center;
  gap: 32px;

  // Button should be the last element of the form.
  // Once button component has the ability to select the large option this should
  // no longer be needed.
  > *:last-child {
    height: 40px;
    padding: 16px;

    div {
      text-transform: none;
      margin: 0;
    }
  }
}
```